test/e2e/suites/e2e/e2e_test.go

```diff
@@ -483,6 +487,7 @@ func makeOpenStackMachineTemplateWithPortOptions(namespace, clusterName, name st
 						Name: fmt.Sprintf("%s-cloud-config", clusterName),
 					},
 					Ports: *portOpts,
+					Tags:  machineTags,
```

Comment:
Oh, we really can't tag ports individually? I think we're going to have to fix that 🤔